docs/src/background/other_projects.rst

```diff
@@ -0,0 +1,360 @@
+- User community: small.
+
+Like other projects on this page, Cinch :cite:`cinch` is designed to cut back
+on the amount of coding needed to write a CMake-based build system. The
+documentation is a bit sparse, but it appears that Cinch provides CMake bindings
+that wrap a Python tool
+`cinch-utils <https://github.com/laristra/cinch-utils>`_. The build system
+developer then writes a CMake-based build system in terms of the CMake bindings.
+
+In terms of features, Cinch seems to be primarily interested in support
+facilitating the building of libraries and executables as well as unit tests
+and documentation for the libraries and executables. Cinch seems to assume that
+the source tree also contains the dependencies' source, and relies on recursive
+builds to create the final package. Additional package management appears to be
+limited to calling ``find_package``. Finally, Cinch also contains a seemingly
+out of place C++ logging system. The latter in particular makes this project
+feel like it was targeting a particular group's workflow, rather than being
+meant as general tool.
+
+*******
+CMake++
+*******
+
+- Website: N/A.
+- Source: `<https://github.com/toeb/cmakepp>`_
+- Development status: inactive.
+- User community: large.
+
+CMake++ :cite:`cmakepp` is a tour-de-force of what is possible with the
+traditional CMake
+language. At its core, CMake++ is meant to be more of a library for CMake,
+then a build system. That said it does contain a number of features which can
+be used to simplify writing build systems including native dependency
+management support. While CMake++ would have been an excellent starting point
+for CMaize, the CMake++ project has been abandoned and lacks documentation
+(except for high-level functionality); in turn complicating the process of
+resurrecting CMake++.
+
+*********
+cmake-get
+*********
+
+- Website: N/A
+- Source: `<https://github.com/pfultz2/cmake-get>`_
+- Development status: inactive.
+- User community: small.
+
+cmake-get :cite:`cmake_get` is a CMake module designed to be a
+configuration-time replacement for  CMake's ``ExternalProject`` module.
+Thus cmake-get simplifies the process of obtaining and building dependencies,
+but not the process of defining a CMake build system.
+
+**********
+cmake-init
+**********
+
+- Website: `<https://github.com/friendlyanon/cmake-init/wiki/Examples>`_
+- Source: `<https://github.com/friendlyanon/cmake-init>`_
+- Development status: active.
+- User community: large.
+
+cmake-init :cite:`cmake_init` is a tool for generating CMake-based build
+systems. cmake-init is specifically designed to help setup CMake build systems
+which can be consumed by the ``FetchContent`` module. Like many other tools on
+the list, cmake-init is designed with a certain stack in mind, though that
+stack is pretty standard (e.g., clang-tidy, Doxygen, clang-format). Like other
+generators, cmake-init suffers from the problems described
+:ref:`here <why_not_a_generator>`.
```

Comment:
```suggestion
generators, cmake-init suffers from the problems described in
":ref:`why_not_a_generator`".
```
Same as my above comment about letting the section name be filled in for "Why not a generator?".